config/flags.go

```diff
@@ -8,5 +8,6 @@ import "github.com/spf13/pflag"
 func BuildFlagSet() *pflag.FlagSet {
 	fs := pflag.NewFlagSet("awm-relayer", pflag.ContinueOnError)
 	fs.String(ConfigFileKey, "", "Specifies the relayer config file")
+	fs.BoolP(VersionKey, "v", false, "Display awm-relayer version")
```

Comment:
```suggestion
	fs.BoolP(VersionKey, "", false, "Display awm-relayer version")
```

Let's only allow the long form here to prevent confusion over `-v` which is commonly used for `--verbose`.